terraso_backend/apps/shared_data/permission_rules.py

```diff
@@ -29,6 +29,11 @@ def allowed_to_delete_data_entry(user, data_entry):
     )
 
 
+@rules.predicate
+def allowed_to_add_data_entry(user, group):
+    return user.memberships.approved_only().filter(group__in=[group]).exists()
```

Comment:
```suggestion
    return user.memberships.approved_only().filter(group=group).exists()
```
It seems that the list is redundant here.